mobile/native/src/ln_dlc/mod.rs

```diff
@@ -146,12 +147,13 @@ pub fn run(data_dir: String, seed_dir: String) -> Result<()> {
         )?);
         let node = Arc::new(Node { inner: node });
 
-        runtime.spawn({
+        runtime.spawn_blocking({
             let node = node.clone();
-            async move { node.keep_connected(config::get_coordinator_info()).await }
+            move || async move { node.keep_connected(config::get_coordinator_info()).await }
         });
 
-        runtime.spawn_blocking({
+        // spawn a dedicated thread as this one might be CPU-heavy
+        std::thread::spawn({
```

Comment:
I really like this post, thanks for throwing it in :) 

Because this is something we want to schedule on ongoing basis, IMHO it's not worth throwing it on a threadpool and back.... 
if we could use any of the parallelism in `rayon`, it might have been still worth it, but in this case it seems like an unnecessary abstraction.

I've aligned both sides (and also decreased the wait time to 2s) here: https://github.com/get10101/10101/pull/720

Reply:
I tend to agree that it's hard to imagine us taking advantage of parallelism here.